__mocks__/axios.js

```diff
@@ -0,0 +1,15 @@
+export const successUrl = '/success'
+export const failureUrl = '/failure'
+export const unauthorizedUrl = '/unauthorized'
+export const networkErrorUrl = '/network-error'
+
+export default jest.fn((options) => {
+  const url = options.url
+  const isError = [failureUrl, unauthorizedUrl].includes(url)
+  const status = isError ? 401 : 200
```

Comment:
Given the apps that I've worked on, I would recommend mapping failure to 422 and unauthorized to 400